Libraries/LibWasm/Printer/Printer.cpp

```diff
@@ -536,8 +548,17 @@ void Printer::print(Wasm::Instruction const& instruction)
             [&](Vector<ValueType> const&) { print("(types...)"); },
             [&](auto const& value) { print("(const {})", value); });
 
-        if (instruction.local_index().value())
-            print(" (local index {})", instruction.local_index().value());
+        if (instruction.opcode() == Instructions::local_get
+            || instruction.opcode() == Instructions::local_set
+            || instruction.opcode() == Instructions::local_tee
+            || instruction.opcode() == Instructions::synthetic_argument_get
+            || instruction.opcode() == Instructions::synthetic_local_seti32_const
+            || instruction.opcode() == Instructions::synthetic_i32_storelocal) {
```

Comment:
as above, or at this point, maybe have a specific name for this group of opcodes, lest one forgets one somewhere